bitnami/apache/templates/podmonitor.yaml

```diff
@@ -0,0 +1,39 @@
+{{- if and .Values.metrics.enabled .Values.metrics.podMonitor.enabled }}
+apiVersion: monitoring.coreos.com/v1
+kind: PodMonitor
+metadata:
+  name: {{ template "common.names.fullname" . }}
+  namespace: {{ .Release.Namespace | quote }}
+  labels: {{- include "common.labels.standard" . | nindent 4 }}
+    {{- if .Values.commonLabels }}
+    {{- include "common.tplvalues.render" ( dict "value" .Values.commonLabels "context" $ ) | nindent 4 }}
+    {{- end }}
+    {{- include "common.tplvalues.render" ( dict "value" .Values.metrics.podMonitor.additionalLabels "context" $ ) | nindent 4 }}
+  {{- if .Values.commonAnnotations }}
+  annotations: {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
+  {{- end }}
+spec:
+  podMetricsEndpoints:
+    - port: metrics
+      path: /metrics
+      {{- if $.Values.metrics.podMonitor.interval }}
+      interval: {{ $.Values.metrics.podMonitor.interval }}
+      {{- end }}
+      {{- if $.Values.metrics.podMonitor.scrapeTimeout }}
+      scrapeTimeout: {{ $.Values.metrics.podMonitor.scrapeTimeout }}
+      {{- end }}
+      {{- if $.Values.metrics.podMonitor.relabelings }}
+      relabelings:
+      {{- include "common.tplvalues.render" (dict "value" $.Values.metrics.podMonitor.relabelings "context" $) | nindent 8 }}
+      {{- end }}
+      {{- if $.Values.metrics.podMonitor.relabelings }}
+      metricRelabelings:
+      {{- include "common.tplvalues.render" (dict "value" $.Values.metrics.podMonitor.metricRelabelings "context" $) | nindent 8 }}
```

Comment:
The `$.` shouldn't be required since you're not switching the context, right?

Reply:
right, i have removed $.  .